genesis/process/genesisBlockCreator_test.go

```diff
@@ -273,29 +268,6 @@ func createTrieStorageManagers() map[string]common.StorageManager {
 	return trieStorageManagers
 }
 
-func createRunTypeComponents(t *testing.T) runTypeComponentsHandler {
```

Comment:
Not sure if deleting this is a good idea.
This was added to avoid those long tests failing.
If teamcity passes all checks without it, that's good !

Reply:
long tests are working